modules/node-tuning-operator.adoc

```diff
@@ -23,13 +23,25 @@ ifdef::operators[]
 [discrete]
 == Purpose
 endif::operators[]
-The Node Tuning Operator helps you manage node-level tuning by orchestrating the TuneD daemon. The majority of high-performance applications require some level of kernel tuning. The Node Tuning Operator provides a unified management interface to users of node-level sysctls and more flexibility to add custom tuning specified by user needs.
+The Node Tuning Operator helps you manage node-level tuning by orchestrating the TuneD daemon and achieves low latency performance by using the Performance Addon controller. The majority of high-performance applications require some level of kernel tuning. The Node Tuning Operator provides a unified management interface to users of node-level sysctls and more flexibility to add custom tuning specified by user needs.
```

Comment:
non-blocking: I'd just call it "performance controller" or perhaps "performance tuning controller". But it's pretty late in the game so, again, non-blocking comment.

Reply:
Thanks, I'm trying to verify what is the correct naming convention.

Reply:
Verified the name: Performance Profile controller.